packages/eslint-config/eslint-config.js

```diff
@@ -14,9 +14,22 @@ module.exports = {
       {
         functions: false
       }
-    ]
+    ],
+    '@typescript-eslint/explicit-function-return-type': [
+      'warn',
+      {
+        allowHigherOrderFunctions: true
+      }
+    ],
+    '@typescript-eslint/await-thenable': 'warn',
+    '@typescript-eslint/no-floating-promises': 'error',
+    '@typescript-eslint/restrict-plus-operands': 'error',
+    // TODO: Enable once TS-ESlint is updated
+    // "@typescript-eslint/restrict-plus-operands": "error",
```

Comment:
Redundant - looks like it is already enabled?

Reply:
ah, my mistake. Was supposed to be [`strict-boolean-expressions`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md)